muxpi/parse_image.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+	# Copyright (C) 2019 Luxoft Sweden AB
+	#
+	# Permission to use, copy, modify, and/or distribute this software for
+	# any purpose with or without fee is hereby granted, provided that the
+	# above copyright notice and this permission notice appear in all copies.
+	#
+	# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+	# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+	# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+	# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+	# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+	# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+	# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+	# SOFTWARE.
+	#
+	# For further information see LICENSE
+	#
+	# Script downloads, unzips, compresses(tars)the nightly build depends on architecture and type (qt-auto, minimal),  
+   # copies .tar image to MuxPi. 
+   # Script executes on server. 
+	#
+	# Usage: 
+	# $ ./parse_image.sh [archvariant][DUT IP][neptune/no parameters for minimal]
```

Comment:
Please also update this in the same was as the Usage text for flash.sh.
And this section shouldn't be indented before the '#'

Reply:
Done